process/transactionLog/process.go

```diff
@@ -61,6 +68,30 @@ func (tlp *txLogProcessor) GetLog(txHash []byte) (data.LogHandler, error) {
 	return txLog, nil
 }
 
+// GetLogFromCache retrieves a log generated by a transaction from ram
+func (tlp *txLogProcessor) GetLogFromCache(txHash []byte) (data.LogHandler, bool) {
+	tlp.mut.RLock()
+	defer tlp.mut.RUnlock()
+
+	if txLog, ok := tlp.logs[string(txHash)]; ok {
+		return txLog, true
+	}
+
+	return nil, false
+}
+
+// SaveLogToCache set will a flag with true and txLogProcessor will start save transactions logs also in RAM
+func (tlp *txLogProcessor) SaveLogToCache() {
```

Comment:
I think this should be renamed, the name suggests that it is actually saving the logs, instead it just sets the flag for enabling this functionality

Reply:
done./